pages/subscriptions/signup.js

```diff
@@ -37,8 +37,13 @@ export async function getServerSideProps(ctx) {
       },
     };
   }
+  const response = await fetchGrantDetail(ctx.query);
 
-  return await fetchGrantDetail(ctx.query);
+  if ('redirect' in response) {
```

Comment:
realised after fixing this that we don't even use this anymore, since the one login feature flag will at this point never again be switch off